lib/rules/key-spacing.js

```diff
@@ -197,13 +207,17 @@ module.exports = function(context) {
             }
 
             width = widths[i];
-
-            if (align === "value") {
+            if (isSingleLine(context.getSource(node))) { // Only one line
```

Comment:
Nevermind, you were right of course.  I must have had some other problem in my code when I first tried that.